packages/example/pages/index.tsx

```diff
@@ -263,7 +264,11 @@ export default function Home(): React.ReactElement {
           <div className="mb-2 text-lg">Try it out</div>
           <div className="text-gray-600 mb-4">
             Connect to your wallet of choice and sign something for send a test
```

Comment:
I know you didn't do this but this would be better as
"Connect to your wallet of choice and sign something **to** send a test transaction."
and then maybe a just on the next line
"Example wallet" as a link